src/drivers/imu/amov_imu/amov_imu.cpp

```diff
@@ -0,0 +1,1560 @@
+};
+
+class AMOV_IMU_gyro;
+
+class AMOV_IMU : public device::CDev
+{
+public:
+	AMOV_IMU(device::Device *interface, const char *path_accel, const char *path_gyro, enum Rotation rotation,
+		int device_type, ORB_PRIO priority);
+	virtual ~AMOV_IMU();
+
+	virtual int                 init();
+
+	virtual ssize_t             read(struct file *filp, char *buffer, size_t buflen);
+	virtual int                 ioctl(struct file *filp, int cmd, unsigned long arg);
+
+	void                        task_main();
+        void                        status();
+
+protected:
+	Device                      *_interface;
+
+	friend class                AMOV_IMU_gyro;
+
+	virtual ssize_t             gyro_read(struct file *filp, char *buffer, size_t buflen);
+	virtual int                 gyro_ioctl(struct file *filp, int cmd, unsigned long arg);
+
+private:
+	int                         _device_type;
+	AMOV_IMU_gyro               *_gyro;
+	uint8_t                     _product;	/** product code */
+
+	bool                        _use_hrt;
+
+	struct hrt_call             _call;
+	unsigned                    _call_interval;
+
+	ringbuffer::RingBuffer      *_accel_reports;
+
+	struct accel_calibration_s  _accel_scale;
+	float                       _accel_range_scale;
+	float                       _accel_range_m_s2;
+	orb_advert_t                _accel_topic;
+	int                         _accel_orb_class_instance;
+	int                         _accel_class_instance;
+
+	ringbuffer::RingBuffer      *_gyro_reports;
+
+	struct gyro_calibration_s   _gyro_scale;
+	float                       _gyro_range_scale;
+	float                       _gyro_range_rad_s;
+
+	unsigned                    _sample_rate;
+	perf_counter_t              _accel_reads;
+	perf_counter_t              _gyro_reads;
+	perf_counter_t              _sample_perf;
+	perf_counter_t              _bad_transfers;
+	perf_counter_t              _bad_registers;
+	perf_counter_t              _good_transfers;
+	perf_counter_t              _reset_retries;
+	perf_counter_t              _duplicates;
+	perf_counter_t              _controller_latency_perf;
+
+	uint8_t                     _register_wait;
+	uint64_t                    _reset_wait;
+
+	math::LowPassFilter2p       _accel_filter_x;
+	math::LowPassFilter2p       _accel_filter_y;
+	math::LowPassFilter2p       _accel_filter_z;
+	math::LowPassFilter2p       _gyro_filter_x;
+	math::LowPassFilter2p       _gyro_filter_y;
+	math::LowPassFilter2p       _gyro_filter_z;
+
+	Integrator                  _accel_int;
+	Integrator                  _gyro_int;
+    uint64_t                    _good_trans;
+    uint64_t                    _bad_trans;
+
+	enum Rotation               _rotation;
+
+
+	// last temperature reading for print_info()
+	float                       _last_temperature;
+
+	// keep last accel reading for duplicate detection
+	uint16_t                    _last_accel[3];
+	bool                        _got_duplicate;
+    int                         _task;
```

Comment:
Formatting. Try installing astyle and running `make check_format` locally. Once the CI system gets passed the px4fmu-v2_default build it will still fail on these minor formatting errors.

Reply:
OK, I got it